### PR TITLE
IGAPP-1377: Fix footer hiding zoom buttons

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/components/MapView.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ const StyledIcon = styled(FontAwesomeIcon)<{ direction: string }>`
 const FooterContainer = styled.div`
   position: absolute;
   bottom: 0;
+  max-width: calc(100% - 50px);
```
Comment (sarahsporck) [inline web/src/components/MapView.tsx:63]: Not sure, whether I like the hard coded 50px, but also don't hava a much better idea :). So in my regards you can keep it.

**Review by sarahsporck — APPROVED** [review]
Tested in Firefox.

File: web/src/components/MapView.tsx (hunk 1 above)
Comment (steffenkleinle) [reply web/src/components/MapView.tsx:63]: Me neither but as we don't even set the width of the buttons I figured its okayish...

Comment (LeandraH) [reply web/src/components/MapView.tsx:63]: Agreed, not the greatest but it's okay.

**Review by LeandraH — APPROVED** [review]
Looks good, tested in Firefox

**Pull request merged**